apps/web/src/pages/Pools/PoolAdvertisementPage/components/DataRow.tsx

```diff
@@ -18,31 +18,25 @@ const DataRow = ({ Icon, label, value, suffix }: DataRowProps) => (
       data-h2-margin="base(x1, 0)"
     />
     <div
-      {...(suffix
-        ? {
-            "data-h2-display": "base(grid)",
-            "data-h2-grid-template-columns": "base(1fr) l-tablet(1fr 1fr)",
-            "data-h2-gap": "base(x1)",
-          }
-        : {})}
+      data-h2-display="base(flex)"
```

Comment:
Technically this breaks the other suffixes a bit.  It doesn't look terrible and the latest designs have everything switching to info dialogs soon, anyway.
![image](https://github.com/GCTC-NTGC/gc-digital-talent/assets/8978655/c94b1ca3-2879-44ad-a1f0-69212982e234)